src/azul/chalice.py

```diff
@@ -189,15 +189,30 @@ def _api_gateway_context_middleware(self, event, get_response):
         finally:
             config.lambda_is_handling_api_gateway_request = False
 
+    hsts_max_age = 60 * 60 * 24 * 365 * 2
+
+    response_headers = {
+        'Content-Security-Policy': "default-src 'self'",
+        'X-Content-Type-Options': 'nosniff',
+        'X-Frame-Options': 'DENY',
+        'Referrer-Policy': 'strict-origin-when-cross-origin',
+        'Strict-Transport-Security': f'max-age={hsts_max_age};'
+                                     ' includeSubDomains;'
+                                     ' preload',
+        'X-XSS-Protection': '1; mode=block'
+    }
+
     def _security_headers_middleware(self, event, get_response):
         """
         Add headers to the response
         """
         response = get_response(event)
-        seconds = 60 * 60 * 24 * 365
-        response.headers['Strict-Transport-Security'] = f'max-age={seconds}; includeSubDomains'
-        response.headers['X-Content-Type-Options'] = 'nosniff'
-        response.headers['X-Frame-Options'] = 'DENY'
+        for header in [
+            'Strict-Transport-Security',
+            'X-Content-Type-Options',
+            'X-Frame-Options',
+        ]:
```

Comment:
Now you're duplicating this literal which partially defeats the purpose again.

Furthermore, it is unclear as to why you only specify some of the headers here. It seems like you should segregate the central definition according to purpose so that we don't end up leaking those details into the usage sites. The segregation will make the intent clear.